src/lib/reducers/context.js

```diff
@@ -19,6 +20,10 @@ export const shouldEnableClaudy = state => {
   return !!claudyActions && !!claudyActions.length
 }
 
+export const getWebviewContext = state => {
+  return state.webviewContext
```

Comment:
export const getWebviewContext = state => state.webviewContext;

or

export const getWebviewContext = { webviewContext } => webviewContext;

Reply:
I agree but here I just followed the already existing code style for clarity, even if it's verbose